src/components/Transactions.tsx

```diff
@@ -37,7 +37,7 @@ const Transactions = ({ type, ...transactionDisplayProps }: Props) => {
 
   return (
     <>
-      {transactions.length ? (
+      {transactions?.length ? (
```

Comment:
Can it be `null/undefined` now? Why it wasn't before this PR?

Reply:
It collapsed when I removed the API connections for test reasons. It probably remained - I'll remove it (even though I think that there is no harm for it to be there)

Reply:
I'd rather make sure we use correct typings so that this kind of stuff is not necessary. I'm not against it, just the change seemed a bit random.